ci-operator/step-registry/openshift-qe/installer/azure/ipi/ovn/multi-ipsec/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
It would be better if we linked back to the openshift-qe folder (need multiple ../)

Reply:
Since all the OWNER files are symbolically linked to each other, if we change one, all will change. 
Hopefully I got this correct?